hexanenetlibrarydebug.lua

```diff
@@ -20,6 +28,20 @@ function net.Incoming(len, client)
 			
 		end
 
+        local plySteamid = client:SteamID()
+        --[[ Implement an anti-net spam feature to stop people from attempting to lag the server. ]]--
+
+        antiNetSpam[plySteamid] =  antiNetSpam[plySteamid] or {}
+        antiNetSpam[plySteamid][name] = (antiNetSpam[plySteamid][name] or 0) + 1
+
+        if antiNetSpam[plySteamid][name] > 10 then 
+            hexanenetlibrarydebug.flaggedPlayers[plySteamid] = true
```

Comment:
Nothing is being done with `flaggedPlayers`

Reply:
Resolved in the most recent commit.